documentation/Query-Syntax.md

```diff
@@ -153,7 +209,12 @@ The query evaluator is implemented in the `evaluateExpression` function:
 }
 ```
 
-- `parameters` can also contain one additional field called `connection`, which is an object representing an active database connection. \
+`parameters` can also fields containing active database connection(s), which are required if using one of the database operators (**pgSQL** or **graphQL**).
+
+- For **pgSQL**:  
+  `pgConnection: <node-postgres Client object>`
```

Comment:
```suggestion
  `pgConnection: { query: <function to query the database>}`
```
Based on the change done on #30 the DB is not assessable here, only a IConnection object defined as:
```

export interface QueryResult {
  rows: QueryRowResult[]
}
export interface IConnection {
  query: (text: string, params: any[]) => Promise<QueryResult>
}
```

Reply:
Changed it to:

> `pgConnection: <postGresConnect object>` (or any valid PostGres connection object, e.g. `Client` from `node-postgres`)

Hope this is okay.